duckduckgo/duckduckgo-privacy-extension
# Main popup view tweaks

**Reviewer:** @laurengarcia 



## Description:

Tweaks the appearance of the main popup view a bit. (Ignore the percentage numbers, I've reported that separately.)

![screenshot from 2018-01-12 19-34-43](https://user-images.githubusercontent.com/1431577/34892141-08640f32-f7d0-11e7-8bc4-56e68efbbe5b.png)

## Steps to test this PR:

1. The main popup view should look decent
2. Make sure to check it with both empty and populated top trackers.

## Automated tests:

###### Reviewer Checklist:


###### PR Author Checklist:
